### ISSUE TITLE
[Major Bug] noclip glitch

### ISSUE THREAD

**Issue by BoB-Client** [Thy-Vipe/BeastsOfBermuda-issues#2461]
_Originally written by **Hel | 76561198424440799**_
Game Version: 1.1.1085

*===== System Specs =====
CPU Brand: Intel(R) Core(TM) i5-10300H CPU @ 2.50GHz
Vendor: GenuineIntel
GPU Brand: Intel(R) UHD Graphics
GPU Driver Info: Unknown
Num CPU Cores: 4
===================*
Context: **I was trying to place a leg down and then I glitched in by accident.** 
Map: Rival_Shores

Location: X=-63307.797 Y=49285.246 Z=4392.352

Basically, theres this glitch with velociraptor that allows it to abuse its climbing mechanic to get inside rocks by climbing on specific spots and sliding in, being immune to all predators.
